Doc/source/coordinates.rst

```diff
@@ -98,4 +98,16 @@ different accuracies.  For example, assuming this module has been
 imported as ``spc``, to set the SpacePy backend as the default and set
 ``itol`` to 5 seconds:
 
-    >>> spc.DEFAULTS.set_values(use_irbem=False, itol=5)
\ No newline at end of file
+    >>> spc.DEFAULTS.set_values(use_irbem=False, itol=5)
+
+Note also that the magnetospheric systems listed above require the
+specification of a centered dipole axis. This is specified using the
+first three coefficients of the IGRF, evaluated at the specified time
+(specifically, the g[1][0], g[1][1], and h[1][1] coefficients).
+For the SpacePy backend, this uses the bundled IGRF coefficients file
+in the standard format. The coefficients from a different version of
+the model can be used by placing them in the ``.spacepy`` directory
+with the name ``igrfcoeffs.txt``. This would be used, for example, if
```

Comment:
Based on the code, isn't this the data subdirectory of the .spacepy directory? (And should the reference to the .spacepy directory link configuration.rst, where it's documented?)

Reply:
Yes, and yes. And you might need to remind me how to format the link in docs...

Reply:
```rst
the model can be used by placing them in the ``data`` subdirectory of the :doc:`.spacepy directory <configuration>`
```